kronolith/lib/Kronolith.php

```diff
@@ -1909,7 +1909,14 @@ public static function sendITipNotifications(
         $view->event = $event;
         $view->imageId = $image->getContentId();
 
-        foreach ($event->attendees as $email => $status) {
+        /* Enhancement #3965: Only send updates to organizer if the user is not the organizer */
+        $receivers = $event->attendees;
+        if ($event->organizer && !$this->isUserEmail($event->creator, $event->organizer) ) {
+            /* TODO: Guess organizer name and status if he is one of the attendees */
+            $receivers = array($event->organizer => array('name' => $event->organizer, 'response' => self::RESPONSE_ACCEPTED));
```

Comment:
That would send an acceptence response as soon as an attendee is editing an event that he is not the organizer of. This is not what we want.